Make temp file be generated in a %temp%\NuGet folder. We would want t…

yishaigalatzer opened the pull request:
…o expand on this in a separate set of PRs

Use msbuild with /nologo
Make CommandRunner return CommandRunnerResult
Restore command - Change the timeout to take a flag, and scale to the number of projects in the solution

emgarten commented on src/NuGet.Clients/NuGet.CommandLine/Commands/RestoreCommand.cs:
```diff
@@ -575,9 +578,25 @@ private PackageRestoreInputs DetermineRestoreInputs()
             {
+                int scaleTimeout;
+
+                if (Project2ProjectTimeOut > 0 || projectsWithPotentialP2PReferences.Length == 0)
```
projectsWithPotentialP2PReferences.Length == 0 is handled below

emgarten commented:
:shipit:

Commit:
Fix timeout for msbuild p2p resolution, and some cleanup

Make temp file be generated in a %temp%\NuGet folder. We would want to expand on this in a separate set of PRs
Use msbuild with /nologo
Make CommandRunner return CommandRunnerResult
Restore command - Change the timeout to take a flag, and scale to the number of projects in the solution

yishaigalatzer commented:
merged 0df20a5850dea79917426ca2ee3458dcb11c3059
